### PR TITLE
new version of add testcase for setting/getting a pointer to a long

### PR DESCRIPTION
**Pull request by estebanlm**
trying to fix @zecke conflicts at #3181

### CODE REVIEW

**Comment by Ducasse** [comment]
Esteban you have testBuildingFnSpec – MacOSX32.UnifiedFFI.Tests.Tests.FFICalloutTest

File: src/UnifiedFFI-Tests/FFICalloutTest.class.st [hunk 1]
```diff
@@ -62,13 +62,13 @@ FFICalloutTest >> testBuildingFnSpec [
...
 
-
 	type := generator fnSpec returnType.
 	self assert: type class equals: FFILong.	
 	self assert: type pointerArity equals: 1.
 	
 	self assert: (generator fnSpec arguments size) equals: 14.
-	s := generator fnSpec arguments readStream.
+
+  s := generator fnSpec arguments readStream.
```
Comment (zecke) [inline src/UnifiedFFI-Tests/FFICalloutTest.class.st:112]: looks like a out of place change?

**Comment by Ducasse** [comment]
esteban what do we do with this one?

**Comment by guillep** [comment]
I'll time out this PR on friday

**Comment by zecke** [comment]
Why not rebase and apply (and see if it is fixed with libffi)?

**Comment by guillep** [comment]
@estebanlm is this ok for integration?

**Comment by Ducasse** [comment]
Esteban I will close this PR because it is too old.

**Comment by Ducasse** [comment]
Let me know but we need to clean the PR queue

**Comment by guillep** [comment]
Seems there is no interest for this PR from the author. I'll close. Reopen if needed.

**Comment by estebanlm** [comment]
well, author was not me but @zecke and I didn't completely understood what he was trying to do at the time :(

**Comment by zecke** [comment]
Can you read a long pointer from a struct today?

```
struct foo {
  long* bar
};
```

Either by having a *foo or a value...

**Pull request closed without merging**